<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_appid">
      <property name="text">
       <string>appid</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_appsecret"/>
    </item>
    <item row="0" column="3" rowspan="2">
     <widget class="QPushButton" name="pushButton_connect">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" rowspan="2">
     <widget class="QTableWidget" name="tableWidget">
      <row>
       <property name="text">
        <string>subscribe</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>openid</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>nickname</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>sex</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>language</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>city</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>province</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>country</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>headimgurl</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>subscribe_time</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>groupid</string>
       </property>
      </row>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_appid"/>
    </item>
    <item row="4" column="0" rowspan="2" colspan="2">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_appsecret">
      <property name="text">
       <string>appsecret</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_show_users">
      <property name="text">
       <string>Show Users</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
